hollow/src/test/java/com/netflix/hollow/core/util/RemovedOrdinalIteratorTest.java

```diff
@@ -22,11 +22,11 @@
 
 public class RemovedOrdinalIteratorTest {
 
+    private BitSet previousOrdinals = bitSet(1, 2, 3, 4, 6, 7, 9, 10);
+    private BitSet currentOrdinals = bitSet(1, 3, 4, 5, 7, 8, 9);
```

Comment:
these are not reset on each test (when multiple test cases are ran)

Reply:
They weren't intended to. I failed to make them static final. Now addressed.